packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-case-basic-sign-in/main/index.md

```diff
@@ -4,23 +4,24 @@ title: Basic sign-in flow using the password factor
 
 <ApiLifecycle access="ie" />
 
-> **Note:** In proxy model architectures, where a server-side application using the Embedded SDK is used as a proxy between client applications and Okta servers, a request context for the client applications is required. Security enforcement is expected to be based on the client request context’s IP address and user agent. However, since these values are currently being derived from the server application rather than the client, this enforcement is not available. As a result, network zones or behaviors that drive their conditions based on these request context values (geolocation, IP Address, or user agent) will not work until we can find a solution to the issue.
+> **Caution:** A request context for the client applications is required in proxy-model architectures, where a server-side application uses the Embedded SDK as a proxy between client applications and Okta servers. In this scenario, Okta expects security enforcement to be based on the client request context’s IP address and user agent. Currently these values are being derived from the server application rather than the client, where this security enforcement is not available. As a result, network zones or behaviors that drive their conditions based on these request context values (geolocation, IP Address, or user agent) do not work.
```

Comment:
Consider using `Caution` we the result of an action or statement ends in an error. It's difficult to decipher who sets the expectation for the `security enforcement` but might need to be revised with more clarity about the scenario.